fcrepo-http-api/src/main/java/org/fcrepo/http/api/ContentExposingResource.java

```diff
@@ -602,10 +585,16 @@ protected void replaceResourceWithStream(final FedoraResource resource,
     }
 
     protected void patchResourcewithSparql(final FedoraResource resource,
-                                           final String requestBody,
-                                           final RdfStream resourceTriples)
-            throws MalformedRdfException, AccessDeniedException {
-        resource.updateProperties(translator(), requestBody, resourceTriples);
+            final String requestBody,
+            final RdfStream resourceTriples)
+                    throws MalformedRdfException, AccessDeniedException {
+        if (resource instanceof NonRdfSourceDescription) {
+            // update the description instead
```

Comment:
Thanks, @escowles. I share your understanding. 
1. Given the priorities as you listed, do you see an architectural conflict with the approach presented in this PR?
2. @ajs6f, do you agree with the above noted priorities? and do you see an architectural conflict between the priorities and the approach presented in this PR?
3. Would storing properties on the binary vs. description resource be a more constructive approach?